CHANGELOG_PENDING.md

```diff
@@ -1,6 +1,7 @@
 ### Improvements
 
 - Added references to issues for missing functions in output.
+- Added code generation rename workarounds for pcl keywords.
```

Comment:
```suggestion
- Add references to issues for missing functions in output
- Add code generation rename workarounds for pcl keywords
```

Both of the entries here don't conform to the chanagelog style we adopted in pulumi/pulumi.  We should probably be consistent with that here as well.  See https://github.com/pulumi/pulumi/blob/master/CONTRIBUTING.md#changelog-messages.